### PR TITLE
DOCS-467 - Copilot Preview 2

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Copilot Preview 2, pending approval from PM (@banant) and tagged devs.

No release notes.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/DOCS-467

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208296810874771

### CODE REVIEW

File: docs/search/copilot.md [hunk 1]
```diff
@@ -110,83 +110,72 @@ In this section, you'll learn the recommended workflow for using Copilot effecti
...
+<img src={useBaseUrl('img/search/copilot/chart-types.png')} alt="Copilot chart types" style={{border: '1px solid gray'}} width="500" />
 
 #### Edit query code
 
 If needed, you can edit your log search query code.
 
-1. Click **Show Log Query** to show the current investigation as a log query.<br/><img src={useBaseUrl('img/search/copilot/show-hide-query.gif')} alt="Copilot time period" style={{border: '1px solid gray'}} width="500" />
-1. Click in the code editor field and edit your search. Not familiar with Sumo Logic query language? See [Search Query Language](/docs/search/search-query-language) to learn more.<br/><img src={useBaseUrl('img/search/copilot/code-editor.png')} alt="Copilot time period" style={{border: '1px solid gray'}} width="600" />
+1. Click in the code editor field and edit your search. Not familiar with Sumo Logic query language? See [Search Query Language](/docs/search/search-query-language) to learn more.<br/><img src={useBaseUrl('img/search/copilot/code-editor.png')} alt="Copilot time period" style={{border: '1px solid gray'}} width="500" />
+   <!-- These two notes conflict. I've asked for clarity.
```
Comment (kimsauce) [inline docs/search/copilot.md:172]: [Pending Slack response](https://sumologic.slack.com/archives/C06MGHMLKJ8/p1727586126038479)

File: docs/search/copilot.md [hunk 2]
```diff
@@ -21,169 +21,100 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 
-<img src={useBaseUrl('img/search/copilot/copilot-tab.png')} alt="Copilot tab" style={{border: '1px solid gray'}} width="200" />
+From the [**Classic UI**](/docs/get-started/sumo-logic-ui-classic), navigate to the **Copilot** tab on the Sumo Logic home page.<br/><img src={useBaseUrl('img/search/copilot/copilot-tab.png')} alt="Copilot tab" style={{border: '1px solid gray'}} width="200" />
+
+From the [**New UI**](/docs/get-started/sumo-logic-ui), click **Copilot** in the left nav.<br/><img src={useBaseUrl('img/search/copilot/copilot-tab-new.png')} alt="Copilot tab" style={{border: '1px solid gray'}} width="150" />
 
 ### Step 2: Select a source category
 
-Click **Select Source Category** - the source expression box - and type/select the data source of the log messages you want to investigate.
+Click **Select Source Category** - the source expression box - and type/select the data source of the log messages you want to investigate. In this example, we'll do AWS WAF.
```
Comment (jpipkin1) [inline docs/search/copilot.md:59]:
```suggestion
Click **Select Source Category** - the source expression box - and type/select the data source of the log messages you want to investigate. In this example, we'll select a source for AWS WAF.
```

File: docs/search/copilot.md [hunk 3]
```diff
@@ -195,27 +126,58 @@ If needed, you can edit your log search query code.
...
+
+#### New Conversation
+
+To start over and begin a new investigation, click **New Conversation**. <br/><img src={useBaseUrl('img/search/copilot/new-conversation.png')} alt="Copilot new conversation" style={{border: '1px solid gray'}} width="700" />
+
 
 ### Step 5: Open in Log Search
 
-Click the **Open in Log Search** icon (insert pic), which will copy your query from Copilot over to a new Log Search, allowing you to utilize all of Sumo Logic's search functionality. You can continue investigating, save the search, and remediate.
+Click the **Open in Log Search** icon, which will copy your query from Copilot over to a new Log Search, allowing you to utilize all of Sumo Logic's search functionality. You can continue investigating, save the search, and remediate.
```
Comment (jpipkin1) [inline docs/search/copilot.md:152]:
```suggestion
Click the **Open in Log Search** icon, which will copy your query from Copilot over to a new log search, allowing you to utilize all of Sumo Logic's search functionality. You can continue investigating, save the search, and remediate.
```

**Review by jpipkin1 — COMMENTED** [review]
Wow. This is super fantastic, @kimsauce! I have only a couple of tiny suggestions.

**Review by jpipkin1 — APPROVED** [review]

**Comment by kimsauce** [comment]
> Wow. This is super fantastic, @kimsauce! I have only a couple of tiny suggestions.

Thank you!!

File: static/img/search/copilot/refine1.png
Comment (AnvitagGarg) [inline static/img/search/copilot/refine1.png:1]: I think, we are planning to remove refine section @banant

Comment (banant) [reply static/img/search/copilot/refine1.png:1]: yes

Comment (kimsauce) [reply static/img/search/copilot/refine1.png:1]: Thanks for the heads up! Is this removal reflected on staging yet?

**Pull request merged**